Add info about People Chain (Kusama) (#5912)
w3f/polkadot-wiki: docs/learn/learn-guides-identity.md

[Polkadot Support Team](https://support.polkadot.network/support/home). For more user-friendly tools
see the [wallets](./wallets-index), [apps](./apps-index) and [dashboard](./dashboards-index) pages." />

:::warning The identity pallet is no longer on the Kusama Relay Chain.

If you are on **Kusama**, any of the extrinsics which require you to use the relay chain now have to
be called via the system parachain,
[which you can find here.](https://polkadot.js.org/apps/?rpc=wss://kusama-people-rpc.polkadot.io)

> The identity pallet, along with all of its data, has been migrated to and resumes functionality on
> the [People Chain](../general/glossary.md#people-chain), a system parachain for identity management.

Reviewer: ```suggestion
The identity pallet, along with all of its data, has been migrated to
the [People Chain](../general/glossary.md#people-chain), a system parachain which can now be used for identity management.
```
